### PR TITLE
Update gem minor versions

### PR DESCRIPTION
**Pull request by prees1**
- Refactor gemfile grouping
- Update minor versions of gems

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,7 +1,8 @@
 source "https://rubygems.org"
 
-gem 'github-pages', group: :jekyll_plugins
-gem 'jekyll-feed', group: :jekyll_plugins
-gem 'jekyll-redirect-from', group: :jekyll_plugins
-gem 'jekyll-sitemap', group: :jekyll_plugins
-
+group :jekyll_plugins do
+  gem 'github-pages'
+  gem 'jekyll-feed'
```
Comment (pshevtsov) [inline Gemfile:5]: Do we need that `jekyll-feed` gem?

**Review by jmariomejiap — APPROVED** [review]
Thanks 👍

File: Gemfile (hunk 1 above)
Comment (prees1) [reply Gemfile:5]: Good question, I'm not sure. This PR doesn't add or remove any gems, its updating the ones that are here.

Comment (prees1) [reply Gemfile:5]: Yea totally. Is there any issue with merging this PR as is?

We can move the dependency improvements into another PR.

Comment (pshevtsov) [reply Gemfile:5]: No probs. It builds on my machine, so LGTM :smile:

**Review by pshevtsov — APPROVED** [review]

**Pull request merged**